doc/overview/proj4_overview.rst

```diff
@@ -48,7 +48,7 @@ Details
 
   **Website:** @WEB_proj4@
   
-  **Licence:** `X/MIT style Open Source license <httpis://trac.osgeo.org/proj/wiki/WikiStart#License>`_
+  **Licence:** `X/MIT style Open Source license <https://trac.osgeo.org/proj/wiki/WikiStart#License>`_
```

Comment:
Hi @Bakaniko The link
https://trac.osgeo.org/proj/wiki/WikiStart#License
The link is failing it should be:
https://trac.osgeo.org/proj4j#License

(travis is catching the bad links now)